src/components/Dialog/MessageDialog/MessageDialogContentInner.tsx

```diff
@@ -21,7 +22,7 @@ export type BaseProps = {
    */
   subtitle?: React.ReactNode
   /**
-   * ダイアログタイトルの HTML タグ
+   * @deprecated titleTag属性は非推奨です。SectioningContent(Article, Aside, Nav, Section, SectioningFragment)でDialog全体をラップして、ダイアログタイトルのHeadingレベルを設定してください
```

Comment:
nits: `@deprecated` 付けてるので、対処方法だけ案内でよさそう

```suggestion
   * @deprecated SectioningContent(Article, Aside, Nav, Section, SectioningFragment)でDialog全体をラップして、ダイアログタイトルのHeadingレベルを設定してください
```

Reply:
Q.
これ、SectioningContentでDialog全体をラップするのって必須なんでしたっけ？
基本はDialog全体を囲む必要はなくて、Dialog全体のlevelを一段下げたい時だけ囲めば良い？

Reply:
A. はい、おおよそそのとおりですが `Dialog全体のlevelを一段下げたい時だけ` ではなく `Dialog全体のlevelを変更したい場合` にSectioningContentを使ってもらう意図のコメントになっています。
SectioningContent系のコンポーネントにはbaseLevelで基準となるレベルを設定できるため、上げ下げすべてでやってもらえると良さそうと思っています
(そしてこの指摘あったタイミングでbaseLevelがSectioningFragment以外指定できない状態になっていたことに気づいたので追加commitしておきました [2bc9f62](https://github.com/kufu/smarthr-ui/pull/3489/commits/2bc9f62fdf2e6c3716fffbf0722fbab86637538a))

Reply:
uさんのStoryの指摘の方を見て気になったのもあったので、Storyの方だけ直れば良さそう！
https://github.com/kufu/smarthr-ui/pull/3489#discussion_r1250495860